### PR TITLE
GDG buea

### PR DESCRIPTION
**Pull request by xaster-Kies**
Adding the GDG Buea Community

### CODE REVIEW

File: src/res/apps.yaml [hunk 1]
```diff
@@ -601,4 +601,37 @@ items:
     otherlinks:
       - https://cameroun.geo.sm
     image: geosm.jpeg
-#--------------------------------------------------------------
\ No newline at end of file
+#--------------------------------------------------------------
+items:
```
Comment (elhmn) [inline src/res/apps.yaml:605]:
```suggestion
```

Comment (elhmn) [reply src/res/apps.yaml:605]: This needs to be removed, I recommend you to try and run the application locally before submitting the PR

File: src/res/apps.yaml [hunk 2]
```diff
@@ -601,4 +601,37 @@ items:
...
+    #android app store
+    playstore:
+    #ios app store
+    appstore:
+    #slack application
+    slack:
+    #github_account full url
+    github_account: https://github.com/GDG-Buea
+    otherlinks:
+
```
Comment (elhmn) [inline src/res/apps.yaml:635]:
```suggestion
```

**Review by elhmn — CHANGES_REQUESTED** [review]
Thanks for your contribution, it seems like the yaml is broken and the application is not able to parse your yaml file. I left you a comment sharing how I think you could fix it.

**Comment by elhmn** [comment]
@xaster-Kies, what is the status of this PR ? I left you some comments, let me know if you need help addressing them.

**Comment by xaster-Kies** [comment]
Ohh @elhmn sorry. I didnot actually see them. Thanks for tagging me

File: src/res/apps.yaml (hunk 1 above)
Comment (xaster-Kies) [reply src/res/apps.yaml:605]: Ohh ok I see the error. Let me correct that !

**Comment by xaster-Kies** [comment]
Hey @elhmn , I think you can check it out now! So sorry for the delay😅😅😅

**Review by elhmn — APPROVED** [review]
Looks great now, thanks for your contribution. 🙇‍♂️

**Comment by xaster-Kies** [comment]
Thank you very much @elhmn 🙇‍♂️🙇‍♂️

**Pull request merged**